KamerConnect.View.MAUI/Views/ChatPerson.xaml.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using KamerConnect.Models;
+using KamerConnect.Services;
+using KamerConnect.View.MAUI.Pages;
+
+namespace KamerConnect.View.MAUI.Views;
+
+public partial class ChatPerson : ContentView
+{
+    private const string _bucketName = "profilepictures";
+    public ChatPerson()
+    {
+        InitializeComponent();
+        BindingContextChanged += OnBindingContextChanged;
+    }
+    
+    private void OnChatTapped(object sender, EventArgs e)
+    {
+        if (BindingContext is Chat chat)
+        {
+            var chatViewPlaceholder= new ChatViewPlaceholder();
+            chatViewPlaceholder.BackgroundColor = Colors.Red;
+            BindingContext = chatViewPlaceholder;
+            //implement event here that changes sends parameters and change to chatview in the chatpage
```

Comment:
Als je een comment typt voor het nog later doen van dingen Is er een vrij algemene conventie dat je
TODO: ... typt. Dit kon je niet weten en is niet verplicht en hoef je niet aan te passen. Maar dan weet je wel dat dat iets is wat vaak gedaan word om makkelijker je todo comments terug te vinden.